solana/tests/run_mocha_tests.sh

```diff
@@ -33,16 +33,16 @@ git clone \
 	  --filter=blob:none \
 	  https://github.com/wormhole-foundation/wormhole \
 	  wormhole-main > /dev/null 2>&1
-	cd wormhole-main/solana
+	cd wormhole-main/solana/legacy
 	echo "> Building Wormhole Solana bridges"
   DOCKER_BUILDKIT=1 docker build \
 			-f Dockerfile \
 			--build-arg BRIDGE_ADDRESS=agnnozV7x6ffAhi8xVhBd5dShfLnuUKKPEMX1tJ1nDC \
 			-o artifacts-localnet .
-  cd ../..
-  cp wormhole-main/solana/artifacts-localnet/bridge.so $EXISTING_CORE_BRIDGE_BPF
-  cp wormhole-main/solana/artifacts-localnet/token_bridge.so $EXISTING_TOKEN_BRIDGE_BPF
-  cp wormhole-main/solana/artifacts-localnet/nft_bridge.so $EXISTING_NFT_BRIDGE_BPF
+  cd ../../..
+  cp wormhole-main/solana/legacy/artifacts-localnet/bridge.so $EXISTING_CORE_BRIDGE_BPF
+  cp wormhole-main/solana/legacy/artifacts-localnet/token_bridge.so $EXISTING_TOKEN_BRIDGE_BPF
+  cp wormhole-main/solana/legacy/artifacts-localnet/nft_bridge.so $EXISTING_NFT_BRIDGE_BPF
```

Comment:
idk if this even did anything since we clone down from main which doesnt have the legacy namespace